shelter/mrenclaveverifycomand.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"github.com/shelter/utils"
+	"github.com/shelter/verification"
+	"github.com/urfave/cli"
+	"unsafe"
+)
+
+const (
+	defaultsrcurl      = "https://github.com/alibaba/inclavare-containers/"
```

Comment:
naming a golang variable should use camel notation, e.g, defaultSrcUrl.

Reply:
change and unify variables naming style as your comments.